pkg/reconciler/domainmapping/resources/certificate.go

```diff
@@ -32,5 +32,5 @@ import (
 func MakeCertificate(dm *v1alpha1.DomainMapping, certClass string) *networkingv1alpha1.Certificate {
 	certName := kmeta.ChildName(dm.GetName(), "")
 	return routeresources.MakeCertificate(
-		dm, serving.DomainMappingUIDLabelKey, dm.Name, certName, certClass)
+		dm, serving.DomainMappingUIDLabelKey, dm.Name, certName, certClass, "domain.mapping")
```

Comment:
I think this needs to be a valid domain that can answer a challenge, so it should probably be `dm.Name` here.